packages/custom-objects-exporter/readme.md

```diff
@@ -0,0 +1,11 @@
+# custom-objects-exporter
+
+Export [commercetools custom objects](https://docs.commercetools.com/http-api-projects-custom-objects.html) from the commercetools platform.
+
+More info here: https://commercetools.github.io/nodejs/cli/custom-objects-exporter.html
+
+## Install
+
+```bash
+npm i @commercetools/custom-objects-exporter -g
```

Comment:
nitpick: this is inconsistent to the command given in `docs/cli/custom-objects-exporter.md`

Reply:
Please write the `npm install` command in full for posterity's sake ;-)